Detectors/EMCAL/base/include/EMCALBase/Hit.h

```diff
@@ -42,24 +42,24 @@ namespace o2 {
       
       /// \brief Check whether the points are from the same parent and in the same detector volume
       /// \return True if points are the same (origin and detector), false otherwise
-      Bool_t operator==(const Point &rhs) const;
+      Bool_t operator==(const Hit &rhs) const;
       
       /// \brief Sorting points according to parent particle and detector volume
       /// \return True if this point is smaller, false otherwise
-      Bool_t operator<(const Point &rhs) const;
+      Bool_t operator<(const Hit &rhs) const;
       
       /// \brief Adds energy loss from the other point to this point
       /// \param rhs EMCAL point to add to this point
       /// \return This point with the summed energy loss
-      Point &operator+=(const Point &rhs);
+      Hit &operator+=(const Hit &rhs);
```

Comment:
I will need to see - this is a functionality ported back from the code in AliRoot. Probably we will not use the operator + from hits but rather do this at digit level.

For the moment it would be nice if we can keep it as it just provides the operators and doesn't harm. We can remove them again at a later stage.